library/spdm_crypt_lib/fips/libspdm_selftest_hmac.c

```diff
@@ -0,0 +1,78 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2021-2022 DMTF. All rights reserved.
+ *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+ **/
+
+#include "internal/libspdm_crypt_lib.h"
+
+#if LIBSPDM_FIPS_MODE
+
+static const uint8_t key[32] = {0};

```

Comment:
These can all be local to the self-test function. No need to statically allocate them.

Reply:
Thanks. I have move them to local.